### PR TITLE
Add missing subsets to odk config file

### LINKED ISSUE #2422: http://purl.obolibrary.org/obo/uberon/subsets/life-stages-composite.obo has disappeared

### PR DESCRIPTION
**Pull request by anitacaron**
Fixes #2422

### CODE REVIEW

**Review by matentzn — APPROVED** [review]
That's it??? Hahha omg

**Comment by shawntanzk** [comment]
I'm subscribed to uberon cause of shepherding, and saw this PR, and just wanted to add:
HAHAHAHAHHAHAA

File: src/ontology/uberon.Makefile [hunk 1]
```diff
@@ -920,8 +924,8 @@ subsets/subsets/life-stages-mammal.owl: subsets/life-stages-core.owl $(TMPDIR)/d
 
 #subsets/life-stages.obo: uberon.owl
 #subsets/life-stages.obo: composite-metazoan.obo
-subsets/life-stages-composite.obo: composite-metazoan.owl
-	$(OWLTOOLS) $< --reasoner-query -r elk -l 'life cycle stage' --make-ontology-from-results $(URIBASE)/uberon/$@ --add-ontology-annotation $(DC)/description "Life cycle stage subset of uberon composite-vertebrate ontology (includes species stage ontologies)" -o -f obo --no-check $@ --reasoner-dispose 2>&1 > $@.LOG
+subsets/life-stages-composite.owl: composite-metazoan.owl
```
Comment (matentzn) [inline src/ontology/uberon.Makefile:927]: Did you check that both owl and obo are created this way?

Comment (anitacaron) [reply src/ontology/uberon.Makefile:927]: Yes, I have checked it.

**Review by matentzn — APPROVED** [review]

**Pull request merged**